### DISCUSSION TITLE
Is it possible to use AI deck together with the bigquad deck?

### DISCUSSION THREAD

**Discussion by wdliu356** [bitcraze/discussions#1588] [Q&A] [answered]
Is it possible to use AI deck together with the bigquad deck by simply stacking them together?

**Comment by wdliu356** [comment]
I noticed that "CPX uses UART2 to communicate with the ESP32 by default, if that is enabled there will be conflicts". Is there a simple way to solve this problem? It seems that this UART2 is for wireless communication and I need this function so disabling it is not within the option. Is it possible to modify the port on the bigquad deck simply through modifying its driver?

**Reply by tobbeanton** [reply, marked as answer]
Unfortunatly using the BigQuad together with the AI-deck is going to be hard. There is no way to move the ESP32 UART and the BigQuad uses those pins for PWM.
![image](https://github.com/user-attachments/assets/83c7e052-e012-4da2-8e28-05b1fb9f81d6)
It might be possible to do some patching and phsically move the UART, but the is not an easy task. Then it would be much easier to use a [Bolt](https://www.bitcraze.io/products/crazyflie-bolt-1-1/) instead.
